community/releases/release-manager-runbook/index.md

```diff
@@ -0,0 +1,314 @@
+---
+layout: single
+title:  "Release Manager Runbook"
+sidebar:
+  nav: community
+---
+
+{% include toc %}
+
+## Friday before the release window opens
+
+1. Ping [#dev](https://spinnakerteam.slack.com/messages/dev/) reminding everyone
```

Comment:
Now that I look again...
There's only this one step under this heading, so you should probaby just make it a regular paragraph.

Reply:
Updated all the single-step sections to just use paragraphs!